5.machine_learning/README.md

```diff
@@ -0,0 +1,64 @@
+# 5. Machine learning
+
+In this module, we train a logistic regression elasticnet machine learning model to predict if a single cell is either failing or healthy based on the feature selected morphology features from Plate 4. 
+
```

Comment:
I want to give Roshan's README credit as inspo here for the model description part since I wanted to model the level of detail. Thank you for the compliment since I wanted to make sure I described the evaluation and splitting too!

I think I will definitely add a short description of the data and maybe use a table too!